### PR TITLE
xcowsay: add Chinese translation

### PR DESCRIPTION
**Pull request by slightlyskepticalpotat**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

translates the xcowsay page to chinese

### CODE REVIEW

File: pages.zh/linux/xcowsay.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+> 在您的 Linux 桌面上显示一头可爱的牛和指定的消息。
+> 牛的显示时间是固定的或则是根据文本大小计算得出的。 点击牛即马上关闭。
+> 更多信息： <https://www.doof.me.uk/xcowsay/>.
+
+- 显示一头说 “hello, world” 的牛：
+
+`xcowsay "{{hello, world}}"`
+
+- 显示一头有另一个命令的输出的牛：
```
Comment (einverne) [inline pages.zh/linux/xcowsay.md:11]: This translation sounds strange. How about

```suggestion
- 显示一头牛和消息，该消息是另一个命令的输出：
```

File: pages.zh/linux/xcowsay.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+`ls | xcowsay`
+
+- 显示一头有指定 X 和 Y 坐标的牛：
```
Comment (einverne) [inline pages.zh/linux/xcowsay.md:15]:
```suggestion
- 显示一头牛，并且其位置由 X 和 Y 坐标指定：
```

File: pages.zh/linux/xcowsay.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`xcowsay --at={{X}},{{Y}}`
+
+- 显示一头不同大小的牛：
```
Comment (einverne) [inline pages.zh/linux/xcowsay.md:19]:
```suggestion
- 显示一头指定大小的牛：
```

File: pages.zh/linux/xcowsay.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`xcowsay --cow-size={{small|med|large}}`
+
+- 显示思想泡泡而不是说话泡泡：
+
+`xcowsay --think`
+
+- 用不同的照片来代替默认的牛：
```
Comment (einverne) [inline pages.zh/linux/xcowsay.md:27]:
```suggestion
- 用指定的照片来代替默认的牛：
```

**Review by einverne — CHANGES_REQUESTED** [review]
Thank you for the translation. I left some comments for you to take a look at.

File: pages.zh/linux/xcowsay.md (hunk 2 above)
Comment (slightlyskepticalpotat) [reply pages.zh/linux/xcowsay.md:15]: I thought the second sounded more natural, but the first was closer to the [original text](https://github.com/tldr-pages/tldr/blob/main/pages/linux/xcowsay.md).

File: pages.zh/linux/xcowsay.md (hunk 3 above)
Comment (slightlyskepticalpotat) [reply pages.zh/linux/xcowsay.md:19]: Same for this.

File: pages.zh/linux/xcowsay.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`xcowsay --cow-size={{small|med|large}}`
+
+- 显示思想泡泡而不是说话泡泡：
+
+`xcowsay --think`
+
+- 用指定的照片来代替默认的牛：
+
+`xcowsay --image={{path/to/file}}`
```
Comment (kbdharun) [inline pages.zh/linux/xcowsay.md:29]:
```suggestion
`xcowsay --image={{路径/到/文件}}`
```
[Ref](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a suggestion for this page.

File: pages.zh/linux/xcowsay.md (hunk 2 above)
Comment (einverne) [reply pages.zh/linux/xcowsay.md:15]: How about

```suggestion
- 显示一头指定 X 和 Y 坐标的牛：
```

File: pages.zh/linux/xcowsay.md (hunk 3 above)
Comment (einverne) [reply pages.zh/linux/xcowsay.md:19]: I think the cow can only have one size, so the translation you provide may cause confusion. If you have other better options, you can replace it.

File: pages.zh/linux/xcowsay.md (hunk 2 above)
Comment (blueskyson) [reply pages.zh/linux/xcowsay.md:15]: The original sentence is fine.

File: pages.zh/linux/xcowsay.md (hunk 3 above)
Comment (blueskyson) [reply pages.zh/linux/xcowsay.md:19]: The original sentence corresponds to the English page.

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Comment by slightlyskepticalpotat** [comment]
Hi, is it possible to merge this?

**Review by navarroaxel — APPROVED** [review]
sure @slightlyskepticalpotat

**Pull request merged**